core/src/main/scala/kafka/admin/ConfigCommand.scala

```diff
@@ -277,12 +275,10 @@ object ConfigCommand extends Config {
 
     val nl = System.getProperty("line.separator")
     val addConfig = parser.accepts("add-config", "Key Value pairs of configs to add. Square brackets can be used to group values which contain commas: 'k1=v1,k2=[v1,v2,v2],k3=v3'. The following is a list of valid configurations: " +
-            "For entity_type '" + ConfigType.Topic + "': " + nl + LogConfig.configNames.map("\t" + _).mkString(nl) + nl +
-            "For entity_type '" + ConfigType.Broker + "': " + nl + KafkaConfig.dynamicBrokerConfigs.map("\t" + _).mkString(nl) + nl +
-            "For entity_type '" + ConfigType.Client + "': " + nl + "\t" + QuotaConfigOverride.ProducerOverride
-                                                            + nl + "\t" + QuotaConfigOverride.ConsumerOverride + nl +
-            "For entity_type '" + ConfigType.User + "': " + nl + "\t" + QuotaConfigOverride.ProducerOverride
-                                                          + nl + "\t" + QuotaConfigOverride.ConsumerOverride + nl +
+            "For entity_type '" + ConfigType.Topic + "': " + LogConfig.configNames.map("\t" + _).mkString(nl, "\t" + nl, nl) +
```

Comment:
Is it intentional that the LogConfig case has "\t" + nl for the separator, while all the other cases have only nl?